main.py

```diff
@@ -0,0 +1,102 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+from google.cloud import storage
+from collections import OrderedDict
+from operator import getitem
+import cv2
+
+class Image(BaseModel):
+    image_name: str
+
+class Prediction(BaseModel):
+    predictions: dict
+
+app = FastAPI()
+'''
+storage_client = storage.Client()
+'''
+COLORS = [(0, 255, 255), (255, 255, 0), (0, 255, 0), (255, 0, 0)]
+mappings = {
+    "Car": [("Road Problem", 0.5), ("Transportation", 0.3), ("Accident", 0.1), ("Road Maintenance", 0.1)],
+    "Cat": [("Stray Animals", 0.8), ("Volunteering", 0.2)],
+    "Dog": [("Stray Animals", 0.8), ("Volunteering", 0.2)],
+    "Garbage": [("Garbage", 0.8), ("Waste", 0.2)],
+    "Human-face": [("Missing", 0.4), ("Volunteering", 0.6)],
+    "Person": [("Road Problem", 0.2), ("Missing", 0.4), ("Volunteering", 0.5), ("Accident", 0.1)],
+    "Pothole": [("Road Problem", 0.5), ("Transportation", 0.1), ("Accident", 0.1), ("Road Maintenance", 0.3)],
+    "Street-light": [("Road Problem", 0.4), ("Electricity", 0.3), ("Transportation", 0.2), ("Accident", 0.1)],
+    "Traffic-light": [("Road Problem", 0.4), ("Transportation", 0.2), ("Accident", 0.4)],
+    "Traffic-sign": [("Road Problem", 0.4), ("Transportation", 0.1), ("Accident", 0.1), ("Road Maintenance", 0.4)],
+    "bus": [("Road Problem", 0.3), ("Transportation", 0.6), ("Accident", 0.1)],
+    "cyclist": [("Road Problem", 0.5), ("Transportation", 0.4), ("Accident", 0.1)],
+    "pedestrian": [("Road Problem", 0.4), ("Transportation", 0.1), ("Missing", 0.3), ("Accident", 0.2)],
+    "vehicle": [("Road Problem", 0.5), ("Transportation", 0.3), ("Accident", 0.1), ("Road Maintenance", 0.1)]
+}
+'''
+bucket_name = "reportown-347509.appspot.com"
```

Comment:
Can you remove private credentials from here?

Reply:
I will, thank you.